requirements-dev.txt

```diff
@@ -2,3 +2,4 @@
 pandas
 mypy
 pylint
+python-dotenv
```

Comment:
What is this used for?

Reply:
I think it's probably better to use the API key is already in the environment and use `os.getenv` to retrieve it. A startup script could create the API key and add it to the environment for added simplicity/convenience

Reply:
have the API key in the environment variables of the machine running the program, like in my Windows env variables?

Reply:
Yes, you can run `export API_KEY=some_api_key` during the setup up to add the env var for the duration of the current terminal sessions

Reply:
ah yep okay i like that

Reply:
actually just realised that this still means it needs to be on my actual machine, i think i prefer the option of having it in a .env file in the repo which is accessed - i don't like the feeling of clogging up my own machines environment variables for projects that i have going - am i thinking of this wrong?

Reply:
and i could also make the Makefile create a .env file which the user can then fill in

Reply:
It shouldn't clog up the machine because you are only exporting the environment variable temporarily for new processes launched from the current shell session. You can see this by running

```bash
export API_KEY='some_api_key'
echo ${API_Key}
```
Which should print the API key, then in a new shell session run
```bash
echo ${API_KEY}
```
And the value should be unset

Reply:
Okay I understand how I could do that, but wouldn't it still require user input somehow? Otherwise the API key would be in plain text in the setup file, e.g., in the Makefile